src/Compilers/CSharp/Test/Semantic/Diagnostics/OperationAnalyzerTests.cs

```diff
@@ -211,7 +211,7 @@ public void M1()
                 );
         }
 
-        [Fact]
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/8699")]
```

Comment:
> [Fact(Skip = "https://github.com/dotnet/roslyn/issues/8699")] [](start = 7, length = 62)

The referenced bug says it tracks pattern matching switch statements.  This looks like it's being disabled for normal old switch statements.